lib/semantics/canonicalize-do.cc

```diff
@@ -34,16 +34,34 @@ class CanonicalizationOfDoLoops {
             common::visitors{
                 [](auto &) {},
                 // Labels on end-stmt of constructs are accepted by f18 for
```

Comment:
The "for" at the end of this line is left over from the previous version of the comment and should be removed.

Reply:
Thanks, fixed.